lang/cs/Org.Apache.REEF.Common/Protobuf/ReefProtocol/REEFMessageCodec.cs

```diff
@@ -17,12 +17,18 @@
  * under the License.
  */
 
+using Org.Apache.REEF.Tang.Annotations;
 using Org.Apache.REEF.Wake.Remote;
 
 namespace Org.Apache.REEF.Common.Protobuf.ReefProtocol
```

Comment:
Agreed. But maybe not as part of this change, as it is unrelated to the issue resolved by it.